services/src/main/java/org/exoplatform/addons/services/JcrServices.java

```diff
@@ -0,0 +1,91 @@
+package org.exoplatform.addons.services;
+
+import org.exoplatform.addons.entity.FavoriteAcitvity;
+import org.exoplatform.container.ExoContainer;
+import org.exoplatform.container.ExoContainerContext;
+import org.exoplatform.services.jcr.RepositoryService;
+import org.exoplatform.services.jcr.core.ManageableRepository;
+import org.exoplatform.services.jcr.ext.common.SessionProvider;
+
+import javax.jcr.*;
+import javax.jcr.query.Query;
+import javax.jcr.query.QueryManager;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JcrServices {
+    private ExoContainer MyContainer ;
+    private org.exoplatform.services.jcr.RepositoryService RepositoryService;
+    private org.exoplatform.services.jcr.ext.common.SessionProvider SessionProvider ;
+    private ManageableRepository Repository ;
+    private javax.jcr.Session Session ;
```

Comment:
variables should not start with an UpperCase

Reply:
done ! change to lowerCase